superset-frontend/src/components/ErrorMessage/ErrorMessageWithStackTrace.tsx

```diff
@@ -59,6 +60,12 @@ export default function ErrorMessageWithStackTrace({
       );
     }
   }
+  // this is in case an error needs a specific error alert
+  if (error && CUSTOM_ERROR_ALERT_MAP[error.error_type]) {
```

Comment:
maybe add option chaining call here to avoid error:
`CUSTOM_ERROR_ALERT_MAP[error?.error_type]`

Reply:
THat's what I had earlier, but you can't use optional chaining when referencing an object, hence the structure that I have here.